### DISCUSSION TITLE
HO Un problema

### DISCUSSION THREAD

**Discussion by Stefano2100** [aircrack-ng/aircrack-ng#2633] [General]
ciao a tutti, sto riscontrado un problema con la mia schede di rete usb ,dopo averla collegata ad una macchina virtuali kali linux, eseguo il comando "sudo airmong-ng start wlan0" ed effettivamente entra in modalita monitor ma non appena mando il comando "sudo airodump.ng wlan0" mi da il seguente errore "interfacce wlan0:     ioctl(SIOCGFINDEX) failed:No such device.   failed inizialiting wireless card(s): wlan0."     qualcuno puo aiutarmi?

**Comment by Mister-X-** [comment]
The issue is quite clear, the device does not exist, as stated within the error message. You need to use airmon-ng to first determine the right device, then start that one.

It is also very important to read carefully error messages as they often tell what is wrong.

**Reply by Stefano2100** [reply]
grazie mille ho risolto ,sbagliavo il nome proprio dell antenna
